doc/mkdocs/docs/api/basic_json/count.md

```diff
@@ -2,7 +2,7 @@
 
 ```cpp
 template<typename KeyT>
-size_type count(KeyT&& key) const;
+size_type count(KeyT && key) const;
```

Comment:
i think you want const& for a "count" function. Since it shouldn't need to modify the provided key.